apps/bsp/bulk-scan-processor/bulk-scan-processor.yaml

```diff
@@ -14,7 +14,6 @@ spec:
       image: hmctspublic.azurecr.io/bulk-scan/processor:prod-77a26ce-20240717080303 #{"$imagepolicy": "flux-system:bulk-scan-processor"}
       environment:
         STORAGE_BLOB_SELECTED_CONTAINER: ALL
-        PRIVATELAW_ENABLED: true
```

Comment:
This does need to be removed but can't in this PR, it will need to be done after testing and when the other PRs are merged

Reply:
sorting in another pr